src/pages/home/sidebar/ChatSwitcherView.js

```diff
@@ -169,6 +178,9 @@ class ChatSwitcherView extends React.Component {
                         : report.reportName ?? '',
                     reportID: report.reportID,
                     participants,
+                    groupIcons: participants.length >= 2
```

Comment:
this does not seem particualrly correct, we could have 2 participants, but for as per the logic just above `participant1Avatar` or `participant2Avatar` could be `''`, I would check if any of this are empty strings... maybe, or even better since we make similar checks on another place, build a method that gets this  array of 2 avatars based on the particpants, and the personal details.

Reply:
Okay. I think i  understand.
I'm going with checking if any `participant1Avatar` and `participant2Avatar` are empty strings. Mainly cause that's what i understand best.
If you're not okay with this approach, please make the other clearer to me and I'll get it done.  Thanks!